# posX_m[m] posY_m[m] posZ_m[m] yaw_rad[rad] 
# velX_ms[m/s] velY_ms[m/s] velZ_ms[m/s] yaw_rs[rad/s] 
# TIME[s]
# offboardMode:
# 1 - relative position
# 2 - velocity

/mission_supervisor:
  ros__parameters:
    offboardMode: 2
    velX_ms:
    - 0.0
    - 0.0
    - 3.0
    - 3.0
    - 0.0
    - -3.0
    - -3.0
    - 0.0
    velY_ms:
    - 0.0
    - 3.0
    - 3.0
    - 0.0
    - -3.0
    - -3.0
    - 0.0
    - 0.0
    velZ_ms:
    - 5.0
    - 0.0
    - -1.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    yaw_rs:
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    - 0.0
    time_s:
    - 10.0
    - 5.0
    - 10.0
    - 5.0
    - 5.0
    - 10.0
    - 5.0
    - 1.0
    use_sim_time: false